website/docs/r/custom_image.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: "cloudscale"
+page_title: "cloudscale.ch: cloudscale_custom_image"
+sidebar_current: "docs-cloudscale-resource-custom-image"
+description: |-
+  Provides a cloudscale.ch Custom Image resource. This can be used to create, modify, and delete Custom Images.
+---
+
+# cloudscale\_custom_image
```

Comment:
There's probably one escape missing, right?

Reply:
According to [this tool](https://registry.terraform.io/tools/doc-preview) the backslash is probably not even required. But adding it for consistency with other docs.